### PR TITLE
Conversion in static load and based on fuel type

### PR DESCRIPTION
**Pull request by JustinPihony**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/1537)

### CODE REVIEW

File: src/main/scala/beam/agentsim/agents/vehicles/VehicleEnergy.scala [hunk 1]
```diff
@@ -58,6 +58,7 @@ class ConsumptionRateFilterStoreImpl(
   private val milesHeader = "miles"
   private val gallonsHeader = "gallons"
   private val rateHeader = "rate"
+  val conversionRateForJoulesPerMeterConversionFromGallonsPer100Miles = 746.86
```
Comment (wrashid) [inline src/main/scala/beam/agentsim/agents/vehicles/VehicleEnergy.scala]: source, can that be added here?

Comment (JustinPihony) [reply src/main/scala/beam/agentsim/agents/vehicles/VehicleEnergy.scala]: I don't follow what you're asking.

File: src/main/scala/beam/agentsim/agents/vehicles/VehicleEnergy.scala [hunk 2]
```diff
@@ -79,38 +88,50 @@ class ConsumptionRateFilterStoreImpl(
           settings.setHeaderExtractionEnabled(true)
           settings.detectFormatAutomatically()
           val csvParser = new CsvParser(settings)
-          loadConsumptionRatesFromCSVFor(filePath, csvParser)
+          vehicleType.primaryFuelType == FuelType.Electricity
```
Comment (wrashid) [inline src/main/scala/beam/agentsim/agents/vehicles/VehicleEnergy.scala]: what is rational behind this?

Comment (JustinPihony) [reply src/main/scala/beam/agentsim/agents/vehicles/VehicleEnergy.scala]: This was left in during testing...it's a NOP, but I removed it

**Comment by JustinPihony** [comment]
test!

**Pull request merged**